packages/web-frontend/src/containers/TopBarLogo/index.js

```diff
@@ -29,8 +29,13 @@ const LogoWrapper = styled.div`
 `;
 
 const LogoImage = styled.img`
-  max-height: 80%;
+  max-height: 40px; // 80% of top bar height
```

Comment:
I'm not too concerned but could we use calc based on the top bar height variable for these heights?